docs/accordproject.md

```diff
@@ -47,7 +49,7 @@ To make sense of the data, a _Data Model_, expressed in the Concerto schema lang
   o Double monthlyPayment // monthlyPayment is a floating-point number
 ```
 
-The Data Types allow a computer to validate values inserted into each of the `{{variable}}` placeholders (e.g., `2.5` is a valid `{{rate}}` but `January` isn't). In other words, the Data Model lets a computer make sense of the structure of (and data in) the clause. To learn more about Data Types see [Concerto Modeling](model-concerto).
+The Data Types allow a computer to validate values inserted into each of the `{{variable}}` placeholders (e.g., `2.5` is a valid `{{rate}}` but `January` isn't). In other words, the Data Model lets a computer make sense of the structure of (and data in) the clause. To learn more about Data Types see [Concerto Overview](model-concerto).
```

Comment:
The whole Section is called `Concerto Modeling`, only the first page is called `Concerto Overview`